runtime/vm/jvminit.c

```diff
@@ -2057,7 +2057,11 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 			 * defer initialization of network data until after heap allocated, since the intialization
 			 * can initiate DLL loads which prevent allocation of large heaps.
 			 */
-			populateRASNetData(vm, vm->j9ras);
+			argIndex = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXREADIPINFOFORRAS, NULL);
+			argIndex2 = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXNOREADIPINFOFORRAS, NULL);
+			if ((argIndex > argIndex2) || (argIndex2 < 0)) {
```

Comment:
`(argIndex >= argIndex2)` should work. If neither options are found they will both be -1.